### PR TITLE
Add a workflow to the docs explaining how to retain the value of renamed exported properties

### PR DESCRIPTION
**Pull request by AdriaandeJongh**
This PR is the result of a discussion I had with @AThousandShips on proposal https://github.com/godotengine/godot-proposals/issues/10644. All feedback is welcome.

### CODE REVIEW

File: tutorials/scripting/gdscript/gdscript_exports.rst [hunk 1]
```diff
@@ -427,3 +427,42 @@ described in :ref:`doc_accessing_data_or_logic_from_object`.
...
+can be applied.
+
+1. Close any scenes or other resources where the exported property is saved to.
+2. Add the ``@tool`` annotation to the script with the exported property you want to rename.
+3. Remember the 'old' name of the exported property and rename it.
+4. Add a ``_set(property, value)`` function to the script and check if its parameter 
+   ``property`` equals the name of the old exported property, in which case: set the new 
+   property to ``value``. Save the script.
+5. Open or load all scenes or other resources where the exported property is saved to and 
+   re-save them. The properties should have successfully transfered!
```
Comment (MaaaxiKing) [inline tutorials/scripting/gdscript/gdscript_exports.rst:445]:
```suggestion
   re-save them. The properties should have successfully transferred!
```

**Review by MaaaxiKing — CHANGES_REQUESTED** [review]
typo

File: tutorials/scripting/gdscript/gdscript_exports.rst [hunk 2]
```diff
@@ -427,3 +427,42 @@ described in :ref:`doc_accessing_data_or_logic_from_object`.
...
+
+Retaining the value of renamed exported properties
+--------------------------------------------------
+
+When you change the name of an exported property, the value it held is lost by default. 
+In order to transfer the value from the old property to the new, the following workflow 
+can be applied.
+
+1. Close any scenes or other resources where the exported property is saved to.
+2. Add the ``@tool`` annotation to the script with the exported property you want to rename.
```
[the remark this answers is not in the document]
Comment (AdriaandeJongh) [reply tutorials/scripting/gdscript/gdscript_exports.rst:439]: Hmm, I hear you. These two scenarios are very suboptimal and potentially a lot of work, so they do make me circle back to my original proposal of adding a much more user-friendly `@exported_previously_as('<property_name>')` annotation to simply 'make it work' for users. In my 12 years of game dev making 10+ games, I've found that refactoring is an important and very often used tool in game development, and so anything to soften the blow would help. 

@dalexeev do you think I should expand this PR with your additional methods / workarounds / workflows, or do you think the challenges of refactoring are addressed with other features on the timeline and these additions to the docs will be out of date soon?

**Comment by AdriaandeJongh** [comment]
This isn't a good workaround / solution after all, as explained in @dalexeev's review. I'm hoping a better workflow can be proposed and implemented.

**Pull request closed without merging**